opm/core/props/phaseUsageFromDeck.cpp

```diff
@@ -135,7 +135,7 @@ PhaseUsage phaseUsage(const Phases& phases)
     pu.has_zFraction = phases.active(Phase::ZFRACTION);
     if (pu.has_zFraction) {
         pu.phase_pos[BlackoilPhases::ZFraction] = numActivePhases;
-        ++ numActivePhases;
+        //++ numActivePhases;
```

Comment:
If it is unnecessary it is unnecessary - delete!

Reply:
> If it is unnecessary it is unnecessary - delete!

This one stays.  I don't care if it's unnecessary or if static analyzers warn about it.  It makes the code easier to maintain if we add new enumerators to `BlackoilPhases`.

Reply:
so you want me to drop the commit?

Reply:
Just to be clear: I want this particular change reverted and the increment reinstated.

Reply:
Just to be clear: I want this particular change reverted and the increment reinstated.